README.md

```diff
@@ -1 +1,558 @@
 # S4 - Server Side Swift Standards
+
+## Mission
+
+It is Swift X's mission to create concise, clear, and well-documented standards to which all Swift projects can conform. By standarzing certain common functionalities, Swift X allows for radically improved cross-project compatibility for anything Swift. More compatibility means more collaboration which benefits each individual project as well as the Swift community as a whole.
+
+Swift X strives to maintain the following core beliefs for all of its standards:
+- Unonpinionated
+    - Swift X does not want to provide any opinion as to how your project should be structured or how it should function. Our only purpose is to facilitate cross-project communication.
+- Performant
+    - Conforming to Swift X standards should have the minimum possible impact on performance possible.
+- Minimum viable requirements
+    - Only the code that is absolutely necessary for cross-project compatibility will be included. 
+- Best practices
+    - Swift X should abide by any widely accepted best practices in the Swift community, like naming conventions. 
+
+## Byte
+
+The `Byte` type represents a byte.
+
+```swift
+public typealias Byte = UInt8
+```
+
+### Motivation
+
+`Byte` is clearer than `UInt8` in the context of binary data.
+
+## Data
```

Comment:
I think we might want to remove this section in the interim since there's a lot of opinions, it likely warrants its own thread.

Also conflict w/ `typealias Data = [Byte]`